### ISSUE TITLE
Add a page to allow editing of a transaction before re-running

### ISSUE THREAD

**Issue by rcrichton** [jembi/openhim-console#24]

**Comment by BMartinos** [comment]
Are we still implementing this feature? if so, should only single rerun transactions be able to configure the rerun settings? It makes it a lot more complicated changing settings for multiple reruns as they could have different channels

**Comment by rcrichton** [comment]
I think this feature needs a lot of thought as it could be difficult to implement correctly. We don't really have an implementation that wants to use this for now so I don't think we need to implement it for now. However, I agree this would probably only be for single re-runs.

**Comment by BMartinos** [comment]
I agree this should be for a single re-run as multiple rerun across diff channels makes it far more complex and I'm not sure what that use case could be. 

The concept i had for the rerun custom settings it to load a popup modal with a form to capture the new rerun settings. The rerun settings can be saved to a new property in the transactions model ( rerun_settings ). When the transactions are being picked off the worker que we can query the rerun_settings property and apply accordingly. 

Once the rerun has been done we can remove these "rerun_settings" to restore the original transactions to its original state or keep these settings in case the transaction needs to be rerun again.
